indigo_drivers/aux_ppb/indigo_aux_ppb.c

```diff
@@ -18,6 +18,7 @@
 
 // version history
 // 2.0 by Peter Polakovic <peter.polakovic@cloudmakers.eu>
+// 2.0.1 Added support for Pocket Powerbox Advance
```

Comment:
this is not version 2.0.1 you can just add date your name and what is changed

however you should increase the driver version 
#define DRIVER_VERSION 0x0011 -> #define DRIVER_VERSION 0x0012